src/lib/site-import/status.ts

```diff
@@ -351,12 +384,11 @@ ${ maybeExitPrompt }
 						commandOutput: failedImportStep.output,
 						error: 'Import step failed',
 						stepName: failedImportStep.name,
-						errorText: failedImportStep.error,
```

Comment:
I had to drop `errorText`:
1. It is not used in `getErrorMessage()`
2. `AppEnvironmentStatusProgressStep` does not have the `error` field :-(